<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Produto</class>
 <widget class="QDialog" name="Produto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Produtos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelCodigo">
       <property name="text">
        <string>Código:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtCodigo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelDescricao">
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtDescricao"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelValor">
       <property name="text">
        <string>Valor:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtValor"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelEstoque">
       <property name="text">
        <string>Estoque:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtEstoque"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelEstoqueMin">
       <property name="text">
        <string>Estoque mínimo:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtEstoqueMin"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnNovo">
       <property name="text">
        <string>Novo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSalvar">
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeletar">
       <property name="text">
        <string>Deletar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tabelaProdutos"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
